operational/azure/azure_long_running_instances/azure_long_running_instances.pt

```diff
@@ -0,0 +1,279 @@
+name "Azure Long Running Instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks for running instances that have been running longer than the `Days Old` parameter. See the [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "low"
+category "Operational"
+info(
+    version: "2.0",
```

Comment:
In the change log version 1.0, both need to be same.

Reply:
You're absolutely right! I've fixed that error. Thank you!